src/main/java/uk/gov/pay/api/clients/ConnectorClient.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.pay.api.clients;
+
+import org.slf4j.MDC;
+
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.Response;
+import java.util.Map;
+import java.util.Optional;
+
+public class ConnectorClient {
+
+    private final Client client;
+
+    @Inject
+    public ConnectorClient(Client client) {
+        this.client = client;
+    }
+
+    public Response get(String url) {
+        MultivaluedMap<String, Object> headers = getHeaders();
```

Comment:
would be good to be consistent with the `post` method as to whether we inline the `getHeaders` call or not